src/enqueue/class-localization-constants.php

```diff
@@ -219,6 +243,20 @@ protected function add_mailchimp_constants( array $localization ): array {
     return $localization;
   }
 
+  /**
+   * Localize all constants required for Mailerlite integration.
+   *
+   * @param  array $localization Existing localizations.
+   * @return array
+   */
+  protected function add_mailerlite_constants( array $localization ): array {
```

Comment:
If the class isn't used for extending, all of these should be `private` not `protected`.

https://medium.com/@ivorobioff/keep-it-private-unless-you-guarantee-to-protect-it-1b95382c1961